### DISCUSSION TITLE
importing the roster

### DISCUSSION THREAD

**Discussion by LurhakongolaB** [community/community#121663] [New to GitHub]
Hi, kindly assist me, this is my first experience using GitHub Classroom. My big issue is that I am getting errors as I am trying to import the roster from Google Classroom.  this the error message "404
Page not found
The page you are trying to reach does not exist, or has been moved.
[Go back to classroom dashboard](https://classroom.github.com/)"

**Comment by briancasio** [comment]
Hi, I'm getting the same error, did you find an answer?
